### PR TITLE
Added symlink resolution for host path and windows platform support in sandbox-platform

### PR DESCRIPTION
**Pull request by SaswatB**

### CODE REVIEW

File: pkg/server/service_windows.go [hunk 1]
```diff
@@ -53,14 +55,16 @@ func (c *criService) getDefaultSnapshotterForSandbox(cfg *runtime.PodSandboxConf
 		platform string
 	)
 	if cfg != nil {
-		platform, _ = cfg.Labels["sandbox-platform"]
+		platform = strings.Replace(cfg.Labels["sandbox-platform"], "-", "/", -1)
```
Comment (jterry75) [inline pkg/server/service_windows.go]: This change is urelated right? Why do we need to take it?

Comment (SaswatB) [reply pkg/server/service_windows.go]: I added two changes in this pr, as per the title. If you'd rather not have that change I can leave it out.

Comment (SaswatB) [reply pkg/server/service_windows.go]: oh wait, I guess this is a third change. it's need to let kubelet specify the label as it doesn't allow /

Comment (jterry75) [reply pkg/server/service_windows.go]: This is a beaking change right now. Can we do this a formal change

**Pull request merged**